data/abilities.ts

```diff
@@ -4080,10 +4080,10 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 		num: 260,
 	},
 	victorystar: {
-		onAllyModifyMove(move) {
-			if (typeof move.accuracy === 'number') {
-				move.accuracy *= 1.1;
-			}
+		onFoeModifyAccuracyPriority: -1,
+		onFoeModifyAccuracy(accuracy) {
```

Comment:
Wait, really? _This_ is why Victory Star doesn’t work? So it actually has not only been not working,  it’s actually been _lowering_ the teams evasiveness instead? Geez. I think you should probably fix the `ModifyAccuracy` event so that `onAllyModifyAccuracy` works here; it’s not called `ModifyEvasion` after all.

Reply:
And maybe just for fun let’s add a test to make sure allied Victory Star never boosts the opponent’s accuracy.

Reply:
It used to use the `ModifyMove` event, which targets the user of the move, which is why it affects an ally at that point. I'll need to double-check what happens if you target your own ally, so that Victory Star still boosts accuracy in this case; I might need to switch to using `onAnyModifyAccuracy` and check that the attacker and ability holder are on the same side.

Reply:
Yeah, I had to switch to explicitly checking that the attacker is the ability holder or an ally. Should be fixed now.